Fix/text locator heatmap (#195)
Dataport/polar: packages/clients/textLocator/CHANGELOG.md

- Breaking: Adapt client to new backend API. Previous versions are no longer runnable due the backend API change.
- Feature: Implement document search, that is, the AddressSearch bar now also offers results for documents and clicking on one will retrieve a list of toponyms from the backend and resolve them against the gazetteer. Previously, we had a "get all documents regarding place" functionality. Now, a "get all places regarding document" feature is implemented. Additionally to the AddressSearch, this can be triggered on documents found in the previously implemented way; the "get all places regarding document" functionality is available from within the GeoSearch result display.

> - Fix: The heatmap feature has been adjusted to match the new data structure and works again.

Reviewer: ```suggestion
- Fix: Adjust heatmap feature to match the new data structure and have it working again.
```
